src/components/_common/Button/Button.tsx

```diff
@@ -1,3 +1,73 @@
-export const Button = ({ label }: { label: string }) => {
-  return <button>{label}</button>;
+import styled from '@emotion/styled';
+import {
+  BRIGHT_GREY,
+  PRIMARY_BLUE,
+  PRIMARY_BLUE_HOVER,
+  SECONDARY_NAVY,
+  SECONDARY_NAVY_HOVER,
+} from '@/style/colorConstants';
+
+interface ButtonProps {
+  color: 'BLUE' | 'NAVY';
+  isOutlineButton: boolean;
+  width: number;
+  height: number;
+  label: string;
+  handleButtonClick: () => void;
+}
+
+export const Button = ({
+  color = 'BLUE',
+  isOutlineButton = false, //true시 테두리만 있는 버튼
+  width = 200,
+  height = 48,
+  label,
+  handleButtonClick,
+}: ButtonProps) => {
+  const buttonStyle = {
+    width: `${width}px`,
+    height: `${height}px`,
+  };
+
+  return (
+    <StButtonContainer
+      style={buttonStyle}
+      isOutlineButton={isOutlineButton}
+      color={color}
+      onClick={handleButtonClick}>
+      {label}
+    </StButtonContainer>
+  );
 };
```

Comment:
```suggestion
export const Button = ({
  color = 'BLUE',
  isOutlineButton = false, //true시 테두리만 있는 버튼
  width = 200,
  height = 48,
  label,
  handleButtonClick,
  ...props
}: ButtonProps) => {
  const buttonStyle = {
    width: `${width}px`,
    height: `${height}px`,
  };

  return (
    <StButtonContainer
      style={buttonStyle}
      isOutlineButton={isOutlineButton}
      color={color}
      onClick={handleButtonClick}
      {...props}>
      {label}
    </StButtonContainer>
  );
};
```
확장성을 고려해서 추가적인 속성들도 전달할 수 있으면 좋을 것 같습니다!

Reply:
타입스크립트에서도 가능할까요? 나머지 props가 어떤것인지 미정인데 타입으 어떻게 줄 수 있을까요 ?

Reply:
@yejinleee 저 나머지 연산자 쓰니까 에러 없이 되더라구요?

Reply:
오오 그렇네요 ,,,!